barretenberg/cpp/src/barretenberg/ultra_honk/ultra_composer.hpp

```diff
@@ -84,7 +84,7 @@ template <IsUltraFlavor Flavor_> class UltraComposer_ {
 
     ProtoGalaxyProver_<ProverInstances> create_folding_prover(const std::vector<std::shared_ptr<Instance>>& instances)
     {
-        ProtoGalaxyProver_<ProverInstances> output_state(instances, commitment_key);
+        ProtoGalaxyProver_<ProverInstances> output_state(instances);
```

Comment:
I hate the use of the name `output_state` in this and similar functions

Reply:
I have experienced this feeling dozens of times. We must change.